src/Handler/ExceptionHandler.php

```diff
@@ -84,6 +93,10 @@ public function __invoke(
             $handler = $this->handler($type);
             $this->whoops->pushHandler($handler);
 
+            $this->whoops->pushHandler(function($exception) {
+                $this->logger->error($exception);
```

Comment:
I realize this is really late, but couldn't we avoid changing this interface by adding the logging handler in the configuration? There's nothing that specifically needs to have this, other than Whoops, and Whoops is already configured elsewhere.